tests/test_publisher.py

```diff
@@ -0,0 +1,98 @@
+import os
+import queue
+import threading
+from itertools import chain
+
+import pytest
+
+from lektor.publisher import _join_pipes_using_select
+from lektor.publisher import _join_pipes_using_threads
+from lektor.publisher import Command
+
+
+def test_Command_triggers_no_warnings(recwarn):
+    # This excercises the issue where publishing via rsync resulted
+    # in ResourceWarnings about unclosed streams.
+
+    # This is essentially how RsyncPublisher runs rsync.
+    with Command(["echo"], env={}) as client:
```

Comment:
I don't think passing `env={}`  does anything.

Reply:
Probably true.  That's a remnant from when I was trying to elicit the unclosed file warnings within the test.  (See next comment.)  I will delete.